<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simul2D</class>
 <widget class="QMainWindow" name="Simul2D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Particle Registration for 2D Slices</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="margin">
     <number>3</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="handleWidth">
        <number>6</number>
       </property>
       <widget class="myQGraphicsView" name="graphicsView">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="backgroundBrush">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </property>
        <property name="renderHints">
         <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::NonCosmeticDefaultPen|QPainter::TextAntialiasing</set>
        </property>
       </widget>
       <widget class="myQGraphicsView" name="graphicsView2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="backgroundBrush">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </property>
        <property name="renderHints">
         <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::NonCosmeticDefaultPen|QPainter::TextAntialiasing</set>
        </property>
       </widget>
      </widget>
      <widget class="QCustomPlot" name="costPlot" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>300</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuParticles">
    <property name="title">
     <string>Particles</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuParticles"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTest"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>204</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="runPage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>280</width>
          <height>653</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Simulations</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Configuration</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="config">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="plainText">
            <string>Options: +adaptive_sampling +ensemble +intensity
NumberOfParticles: int 100
ForceCoefficients: reals 0.1 0.2 1
PreprocessingTimeRange: reals 0 0.05 30
RunTrace: /tmp/trace.txt
TimeRange: reals 0 0.05 30
InitialIntersectionMaskCache: /tmpfs/mask.nii.gz
Subjects: strings 2
00
11
RealImages: strings 2
/NIRAL/work/joohwi/nadia/SliceImages/C33/C33_E04_slice_128.nii.gz
/NIRAL/work/joohwi/nadia/SliceImages/C31/C31_E04_slice_128.nii.gz
LabelImages: strings 2
/NIRAL/work/joohwi/nadia/SliceImages/C33/C33_E04_label_128.nii.gz
/NIRAL/work/joohwi/nadia/SliceImages/C31/C31_E04_label_128.nii.gz
==option_end==</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0" colspan="3">
            <widget class="QSlider" name="zoom">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="applyButton">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="runStepButton">
             <property name="text">
              <string>Step</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" rowspan="3">
            <widget class="QDial" name="labelOpacity">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="value">
              <number>80</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="loadTrace">
             <property name="text">
              <string>Load Trace</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QPushButton" name="saveTrace">
             <property name="text">
              <string>Save Trace</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="3">
            <widget class="QSlider" name="traceSteps">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="3">
            <widget class="QLCDNumber" name="lcdNumber">
             <property name="styleSheet">
              <string notr="true">QLCDNumber {
    color: rgb(122, 255, 100);    
    background-color: rgb(0, 0, 0);
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="verboseOutput">
           <property name="text">
            <string>Verbose Output</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>17</width>
             <height>245</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tracePage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>280</width>
          <height>653</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Visualization</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="showWarped01">
             <property name="text">
              <string>Show Warped 0 -&gt; 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="showWarped10">
             <property name="text">
              <string>Show Warped 1 -&gt; 0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="printAsPDF">
             <property name="text">
              <string>Print as PDF</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>410</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="optimizerSettings">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>280</width>
          <height>653</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Optimizer Settings</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4"/>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
   <property name="toolTip">
    <string>Print Configurations</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="3"/>
 <customwidgets>
  <customwidget>
   <class>myQGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>myQGraphicsView.h</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
